pkg/networkservice/chains/xconnectns/server.go

```diff
@@ -53,6 +55,7 @@ func NewServer(name string, vppagentCC *grpc.ClientConn, baseDir string, u *url.
 		name,
 		// Make sure we have a fresh empty config for everyone in the chain to use
 		vppagent.NewServer(),
+		directmemif.NewServer(),
```

Comment:
NOTE: I've moved direct memif before 
`memif.NewServer(baseDir)`
Because directmemif should be called after memfi.Server and memif.Client.